projects/quml-library/src/lib/util-service.ts

```diff
@@ -35,4 +35,21 @@ export class UtilService {
         })
     }
 
+    public getMultiselectScore(options, mappings) {
+        return new Promise((resolve, reject) => {
```

Comment:
Why promise is required for synchronous operation, it is unnnessary

Reply:
this is required because without this the score is coming as undefined , it has been tested